Maestro/subscriptions.json

```diff
@@ -241,6 +276,32 @@
         }
       }
     },
+    // Update dependencies in CoreFX release/2.0.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/2.0.0/Latest.txt",
```

Comment:
To clarify `release/1.*.0` do not have these trigger paths. But these additional paths are required for `release/2.0.0`, right?